tests/models/test_gpu.py

```diff
@@ -236,19 +214,19 @@ def test_parse_gpu_returns_none_when_no_devices_are_available(mocked_device_coun
 )
 @mock.patch("torch.cuda.device_count", return_value=1)
 @mock.patch("torch.cuda.is_available", return_value=True)
-@pytest.mark.parametrize("gpus", [[0, 1, 2], 2, "0"])
-def test_torchelastic_gpu_parsing(mocked_device_count, mocked_is_available, gpus):
+@pytest.mark.parametrize("devices", [[0, 1, 2], 2])
```

Comment:
Need acceleartor=gpu

Reply:
i already added acceleartor=gpu in the lines below that